community.md

```diff
@@ -52,7 +52,7 @@ We run an announcement-only, no-marketing-spam mailing list that we will use to
 
 Oh no, you hit a bump on the road! Please tell us - we want to fix all bugs!
 
-- Found a bug in the compiler, standard libraries or the runtime? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">https://github.com/ballerina-platform/ballerina-lang/issues</a>
+- Found a bug in the compiler, standard library or the runtime? Create an issue here: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">https://github.com/ballerina-platform/ballerina-lang/issues</a>

```

Comment:
Should we include IDE tooling also here? @kaviththiranga

Reply:
+1

Reply:
@sanjiva - Shall we add Tooling also here? WDYT?

Reply:
+1 Added tooling as follows:

"Found a bug in the compiler, standard library, runtime, or tooling? Create an <a href="https://github.com/ballerina-platform/ballerina-lang/issues">issue</a>."